apcd-cms/src/apps/registrations/templates/submission_form/submission_form.html

```diff
@@ -1990,12 +1990,12 @@ <h6>
     <p>
       <small
         >¹ Third Party Administrator / Administrative Services Only (TPA/ASO)<br />
-        ² Federal Employer Identification Number (FEIN)<br />
-        ³ National Association of Insurance Commissioners (NAIC)<br />
+        ² Secure File Transfer Protocol (SFTP)<br />
+        ³ Hypertext Transfer Protocol Secure (HTTPS)<br />
         ⁴ United States Dollar&nbsp;(USD)<br />
-        ⁵ Secure File Transfer Protocol (SFTP)<br />
-        ⁶ Hypertext Transfer Protocol Secure (HTTPS)<br />
-        ⁷ Universal Serial Bus (USB)</small
+        ⁵ Universal Serial Bus (USB)<br />
```

Comment:
I think you meant for USB to be 4 and USD to be 5

Reply:
Thank you. 8244fda